test/feature/step_definitions/eval_definitions.coffee

```diff
@@ -3,24 +3,18 @@
 
 module.exports = ->
 
-  @Then /^I have a routing helper "([^"]+)" that returns "([^"]+)"$/, (helper, value, done) ->
-    @makeRequest 'get', "/eval/return String(#{helper})", (err, _, responseBody) ->
-      return done err if err
-      expect(responseBody).to.equal value
-      done()
+  @Then /^I have a routing helper "([^"]+)" that returns "([^"]+)"$/, (helper, value) ->
+    response = yield @makeRequest 'get', "/eval/return String(#{helper})"
+    expect(response.body).to.equal value
 
 
-  @Then /^I have a routing helper "([^"]+)" that returns "([^"]+)" and the HTTP verb "([^"]+)"$/, (helper, value, method, done) ->
-    @makeRequest 'get', "/eval/return String(#{helper})", (err, _, responseBody) =>
-      return done err if err
-      expect(responseBody).to.equal value
-      @makeRequest 'get', "/eval/return String(#{helper}.method)", (err, _, responseBody) ->
-        return done err if err
-        expect(responseBody).to.equal method
-        done()
+  @Then /^I have a routing helper "([^"]+)" that returns "([^"]+)" and the HTTP verb "([^"]+)"$/, (helper, value, method) ->
+    response = yield @makeRequest 'get', "/eval/return String(#{helper})"
+    expect(response.body).to.equal value
+    response = yield @makeRequest 'get', "/eval/return String(#{helper}.method)"
+    expect(response.body).to.equal method
 
-  @Then /^the routing helper "([^"]+)" is undefined$/, (helper, done) ->
-    @makeRequest 'get', "/eval/return String(#{helper})", (err, _, responseBody) ->
-      return done err if err
-      expect(responseBody).to.equal 'undefined'
-      done()
+
+  @Then /^the routing helper "([^"]+)" is undefined$/, (helper) ->
+    response = yield @makeRequest 'get', "/eval/return String(#{helper})"
+    expect(response.body).to.equal 'undefined'
```

Comment:
I'm expecting a string here because of the way this has to work unfortunately. To get fully isolated exprestive tests I call child_process.fork, but that means I had to create an `eval` route to inspect values of things like the routing helpers. I suppose I could create a higher level RPC wrapper around `@makeRequest 'eval/...'` that passes the response to JSON.parse.